README.md

```diff
@@ -54,12 +54,35 @@ Other documentation can be found in [`docs` directory](docs).
 
 ## Support
 
-If you encounter any problems with this release please fill out the
-[PROBLEM-REPORT-FORM](PROBLEM-REPORT-FORM) file found in this directory and use
-it when posting to the [mailing list](http://opendds.org/support.html) or
-creating a [GitHub Issue](https://github.com/objectcomputing/OpenDDS/issues).
+For commercial support, please see https://opendds.org/support.html.
 
-For commercial support please see https://opendds.org/support.html.
+Questions concerning OpenDDS should be directed to [StackOverflow](https://stackoverflow.com/questions/tagged/opendds) or the [mailing list](http://opendds.org/support.html).
+
+If you wish to file a bug report:
+
+1. Fork OpenDDS on GitHub.
+2. Add a [minimum working example](#mwe) that demonstrates the problem.
+3. Create a pull request.
+4. Fill out the [PROBLEM-REPORT-FORM](PROBLEM-REPORT-FORM) and attach it to the pull request.
+
+### <a id="mwe">How to make a minimum working example</a>
+
+1. Create a directory `tests/DCPS/MyExample` (`MyExample` is a placeholder.)
+2. Add the IDL, code, and configuration files for the example programs.  Be sure to exclude any proprietary information.
```

Comment:
```suggestion
2. Add the IDL, code, and configuration files for the example programs.
```

Does it need to be stated that people submitting code to a public repository should be aware of what they're submitting?

Reply:
Yes.  I don't think it hurts anything.